.pre-commit-hooks.yaml

```diff
@@ -27,3 +27,4 @@
   description: "Only check configuration files (TOML/INI/JSON/etc.) and print the violations, according to the Nitpick style"
   entry: nitpick check
   language: python
+  pass_filenames: false
```

Comment:
I had a similar problem before, I'm not sure if changing this line will bring the issue again.

- This was the previous fix removing `pass_filenames`: https://github.com/andreoliwa/nitpick/pull/356
- See this comment: https://github.com/andreoliwa/nitpick/issues/354#issuecomment-821237229
- And also [this comment from the creator of pre-commit himself](https://stackoverflow.com/questions/67104361/make-pre-commit-hooks-ignore-submodules/67110371?noredirect=1#comment118651893_67110371):

> and nitpick run unconditionally checks every file always. they've sidestepped the framework and you should open an issue asking them to change to linting filenames which are passed